### ISSUE TITLE
Some cars can't start race

### ISSUE THREAD

**Issue by cryham** [RigsOfRods/rigs-of-rods#3121]
Recent build: 2024.01-dev.
On: Debian 12, OpenGL, amdgpu.
So I've experienced this on many tracks: Arizona, Auriga, Lone Peak, Outback, f1_testtrack etc.

I can start a race with e.g.
any Gavril, Audi_a4, bmw E39, Daf Semi, Box5 Deuce, dodge magnum, Box5 Gravedigger (all), Rip Saw..
But can't start a race with e.g.
UnlimitedTrophyTruckV2, any from MeanMachine-v1_2, class1buggy10, Box5 W900, Hoodrat..

Any idea what's wrong?
IMO isn't a matter of any sizes or track.
Is anything missing in truck files?

**Comment by CuriousMike56** [comment]
Already reported: https://github.com/RigsOfRods/rigs-of-rods/issues/3097

**Comment by cryham** [comment]
Wow that was fast.
Ok so no solution is known yet

**Comment by cryham** [comment]
Okay so I found that just by adding a `cab` section (I copied it even from other .truck)
at end, to a .track file that didn't have it yet (and couldn't race), did indeed let me race now:
```
submesh
cab
286,287,283, u
283,287,284, u
287,288,284, u
288,285,284, u
283,284,280, u
284,281,280, u
```
But, this isn't really a solution, as this would require editing many .track files.
So perhaps in code there should be a check for that aabb to not be infinte or empty IDK. Or simply if there aren't any cab nodes, picking a few just to have some for aabb.

Strangely I found an old WIP .truck (SDSArgo.zip) with cab section that still didn't start race.
[SDSargo multilift+.txt](https://github.com/RigsOfRods/rigs-of-rods/files/14186232/SDSargo.multilift%2B.txt)
[SDSargo sideloader-.txt](https://github.com/RigsOfRods/rigs-of-rods/files/14186233/SDSargo.sideloader-.txt)
So `SDSargo.sideloader-.truck` can't race, while `SDSargo multilift+.truck` can. Maybe some distance issue, or not important at all.
